File: InsuranceApp/ProcessBankStatementService/Models/BankStatement.cs
namespace ProcessBankStatementService.Models
{
    public class BankStatement
    {
        public int Id { get; set; }
        public string FileName { get; set; }
        public string Error { get; set; }
        public bool Applied { get; set; }
        public string Content { get; set; }
        public DateTime CreatedOn { get; set; }
    }
}

File: Frontend/src/app/models/BankStatement.ts
export class BankStatement {
    id:number;
    fileName: string;
    content: string;
    error: string;
    applied: boolean;
    createdOn: Date;

    constructor(id: number,name: string, content: string, error: string, applied: boolean, createdOn: Date) {
        this.id = id;
        this.fileName = name;
        this.content = content;
        this.error = error;
        this.applied = applied
        this.createdOn = createdOn;
    }
}